### ISSUE TITLE
Changing CTS-URNs from 1st1K to OPP

### ISSUE THREAD

**Issue by AlisonBabeu** [OpenGreekAndLatin/First1KGreek#2592]
While this is a process I've been undertaking as I edit files, I'm opening this issue largely as a placeholder to remind myself and document any issues I find in the process. 

At the beginning of OGL many files that were added using the OPP CTS-URN that had been used to catalog and create CTS-URNs for any non-Perseus editions in the first automatic generation of metadata for the Perseus Catalog in 2013.  Eventually it was decided to have all First1K texts use 1st1K URNs instead, so that some editions have works with OPP and 1st1K URNs in them depending on when the file was added. I've been changing URNs as I work through the Scaife collection see #2573 (for one example) and will continue to link issues here as I find problems etc.

**Comment by AlisonBabeu** [comment]
A related issue is if all the CSEL and other Latin texts should have their URNs changed to OGL from OPP since new Latin texts being added in and viewable on Scaife use OGL not OPP.

**Comment by lcerrato** [comment]
@AlisonBabeu 
Any idea of how many works this affects? In the absence of a redirect function in the software we might not want to break a lot of links of this is really a large amount.
We probably want something in Scaife that will enable notification of retired links or deprecated URNs.
Right now, we don't have a catalog or catalog-function that is going to alert the user to the reassignment so their links will just be dead.
So reassigning 275+ CSEL files as opposed to a few (?) First1K files is on a different scale.

**Comment by AlisonBabeu** [comment]
Hi @lcerrato after doing a quick count I realize that there are actually about 400 Greek OPP URNs still in the 1st1K/Scaife repository as well. I have to admit I don't think anyone is linking to the CSEL texts using their OPP URNS but you are right that we don't have  a way of redirecting or alerting users.  Part of the inspiration for this work isn't just about the intellectual consistency it's that I can't properly catalog any of the CSEL works in the new model (well since 2018, but I digress), since the version in Scaife is a new expression and needs a different URN than the OPP version that already exists in the catalog.

**Comment by lcerrato** [comment]
@AlisonBabeu 
Yes, there isn't a good answer for this. Anyone who is using these corpora for research would encounter a different set of URNs. 
The best way forward is going to be to have some standard notation somewhere — in the headers? (directory? work metadata?) in order to document the changes. I don't know what the best approach would be. Probably a stand off text file in the directory?
